iothub_client/inc/internal/iothub_transport_ll_private.h

```diff
@@ -38,6 +38,7 @@ extern "C"
     typedef void (*pfTransport_Twin_ReportedStateComplete_Callback)(uint32_t item_id, int status_code, void* ctx);
     typedef void (*pfTransport_Twin_RetrievePropertyComplete_Callback)(DEVICE_TWIN_UPDATE_STATE update_state, const unsigned char* payLoad, size_t size, void* ctx);
     typedef int (*pfTransport_DeviceMethod_Complete_Callback)(const char* method_name, const unsigned char* payLoad, size_t size, METHOD_HANDLE response_id, void* ctx);
+    typedef const char* (*pfTransport_GetOption_MODEL_ID_Callback)(void* ctx);
```

Comment:
nit: any reason why `_MODEL_ID_` is all caps?

Reply:
Fixing.